tools/src/h5dump/h5dump.c

```diff
@@ -1357,7 +1357,7 @@ main(int argc, char *argv[])
         goto done;
     }
 
-    /* enable error reporting if command line option */
+    /* Enable error reporting if -E command line option is specified */
```

Comment:
While technically correct, we are discouraging the use of the short names. Use "--enable-error-stack"